aiohttp/client_reqrep.py

```diff
@@ -561,11 +561,8 @@ async def write_bytes(
         """Support coroutines that yields bytes objects."""
         # 100 response
         if self._continue is not None:
-            try:
-                await writer.drain()
-                await self._continue
-            except asyncio.CancelledError:
```

Comment:
write_bytes() becomes the writer task. I'm not clear if there's an easy way to check if cancellation is caused by an outer task awaiting on it or not, so seems simpler to just not suppress the exception at all and handle it in the other code.

There is already exception handling on atleast one case where the task is awaited anyway..